kitsune/products/views.py

```diff
@@ -35,6 +37,16 @@ def product_landing(request, template, slug):
         return HttpResponse(json.dumps({'topics': topic_list}),
                             content_type='application/json')
 
+    if request.LANGUAGE_CODE == settings.WIKI_DEFAULT_LANGUAGE:
+        show_community_support = True
+    else:
+        try:
+            doc = Document.objects.get(locale=settings.WIKI_DEFAULT_LANGUAGE,
+                                       slug='get-community-support')
```

Comment:
I agree, but I am afraid there is no other option how to check, if the article has been translated. It's already done on few other places, this would be the first one outside of templates. Maybe I can rewrite how this is done to be called from the template with the slug.